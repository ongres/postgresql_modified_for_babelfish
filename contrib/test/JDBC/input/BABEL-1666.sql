create procedure p_babel_1666 as
  declare @p1 decimal(5,2);
  set @p1 = 3.14;
  select @p1;
  return @p1;
go

declare @a decimal(5,2);
exec @a = p_babel_1666;
select @a;
go

declare @i int;
exec @i = p_babel_1666;
select @i;
go

drop procedure p_babel_1666
go
