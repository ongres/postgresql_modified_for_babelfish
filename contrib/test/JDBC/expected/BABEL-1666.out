create procedure p_babel_1666 as
  declare @p1 decimal(5,2);
  set @p1 = 3.14;
  select @p1;
  return @p1;
go

declare @a decimal(5,2);
exec @a = p_babel_1666;
select @a;
go
~~START~~
numeric
3.14
~~END~~

~~START~~
numeric
3.00
~~END~~


declare @i int;
exec @i = p_babel_1666;
select @i;
go
~~START~~
numeric
3.14
~~END~~

~~START~~
int
3
~~END~~


drop procedure p_babel_1666
go
